### PR TITLE
[move framework][move-prover] Additional specs, fixed timeouts in Lib…

### PR DESCRIPTION
**Pull request by DavidLDill**
…raSystem and ValidatorConfig

Polishing of specs in LibraSystem and ValidatorConfig.

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: language/stdlib/modules/LibraSystem.move [hunk 1]
```diff
@@ -28,10 +28,10 @@ module LibraSystem {
         validators: vector<ValidatorInfo>,
     }
     spec struct LibraSystem {
-        /// Members of `validators` vector (the validator set) have unique addresses.
-        invariant
-            forall i in 0..len(validators), j in 0..len(validators):
-                validators[i].addr == validators[j].addr ==> i == j;
+       /// Members of `validators` vector (the validator set) have unique addresses.
```
Comment (junkil-park) [inline language/stdlib/modules/LibraSystem.move:31]: The new code does not seem to have the right number of spaces for indentation.

File: language/stdlib/modules/LibraSystem.move [hunk 2]
```diff
@@ -188,13 +190,13 @@ module LibraSystem {
...
         include LibraConfig::ReconfigureAbortsIf;
         aborts_if !spec_is_validator(account_address) with Errors::INVALID_ARGUMENT;
-        ensures !spec_is_validator(account_address);
     }
     spec fun remove_validator {
         let vs = spec_get_validators();
         ensures forall vi in vs where vi.addr != account_address: exists ovi in old(vs): vi == ovi;
-        /// Removed validator should no longer be valid.
-        ensures !spec_is_validator(account_address);
+      /// Removed validator is no longer a validator.  Depends on no other entries for same address
```
Comment (junkil-park) [inline language/stdlib/modules/LibraSystem.move:197]: The new code does not seem to have the right number of spaces for indentation.

File: language/stdlib/modules/LibraSystem.move [hunk 3]
```diff
@@ -217,7 +219,7 @@ module LibraSystem {
         let is_validator_info_updated = update_ith_validator_info_(&mut libra_system_config.validators, to_update_index);
         if (is_validator_info_updated) {
             set_libra_system_config(libra_system_config);
-        }
+        };
```
Comment (junkil-park) [inline language/stdlib/modules/LibraSystem.move:222]: ';' can be omitted if it were ok previously?

**Review by junkil-park — APPROVED** [review]
I left some minor comments and approved.

File: language/stdlib/modules/VASP.move [hunk 4]
```diff
@@ -81,13 +81,25 @@ module VASP {
     spec fun publish_child_vasp_credential {
         /// TODO: this times out some times, some times not. To avoid flakes, turn this off until it
         ///   reliably terminates.
-        pragma verify_duration_estimate = 100;
-        include Roles::AbortsIfNotParentVasp{account: parent};
-        let parent_addr = Signer::spec_address_of(parent);
+//        pragma verify_duration_estimate = 100;
```
Comment (xli) [inline language/stdlib/modules/VASP.move:84]: delete?

**Comment by xli** [comment]
is it intent to change move script and caused compiled bytes changed? 

this is breaking change to client when code is deployed to specific network and causes a client can't connect to different network (testnet, premainnet)

**Comment by DavidLDill** [comment]
I've fixed Junkil's comments.  I accidentally made a bunch of additional changes without creating a new branch, so I have folded these into PR 6240, and I'm closing this one.

**Pull request closed without merging**